synapse/storage/databases/main/cache.py

```diff
@@ -193,7 +193,7 @@ def _invalidate_caches_for_event(
         relates_to: Optional[str],
         backfilled: bool,
     ) -> None:
-        self._invalidate_get_event_cache(event_id)
+        self._invalidate_local_get_event_cache(event_id)
```

Comment:
might be good to add a comment here saying that the shared cache will already have been invalidated, so no need to do it again